.github/workflows/deploy.yml

```diff
@@ -1,16 +1,25 @@
-name: Deploy to GitHub
+name: Deploy Releases
+
 on:
   push:
-    tags:
-      - '*'
+    tags: ["*"]
 
 jobs:
-  deploy:
+  npm:
+    uses: tree-sitter/workflows/.github/workflows/package-npm.yml@main
+    secrets:
+      NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+  crates:
+    uses: tree-sitter/workflows/.github/workflows/package-crates.yml@main
+    secrets:
+      CARGO_REGISTRY_TOKEN: ${{secrets.CRATES_IO_TOKEN}}
+  github:
     runs-on: ubuntu-latest
     env:
-      EMSCRIPTEN_VERSION: '3.1.55'
+      EMSCRIPTEN_VERSION: 3.1.55
```

Comment:
I'd prefer the quoted version here to avoid potential pitfalls with YAML numbers.